app/lib/qualifications_api/client.rb

```diff
@@ -72,11 +72,13 @@ def teachers(date_of_birth:, last_name:)
       raise(QualificationApi::InvalidTokenError) if response.status == 401
 
       results =
-        response.body["results"].map do |teacher|
+        response.body["results"]&.map do |teacher|
```

Comment:
Does this warrant a code comment, given that it's not immediately obvious why the response body wouldn't contain a results field?